tools/shoestring/shoestring/internal/NodeKeyProvider.py

```diff
@@ -0,0 +1,25 @@
+import shutil
+from pathlib import Path
+
+
+class NodeKeyProvider:
+	"""Generators the Node private key."""
+
+	def __init__(self, import_source=None):
+		"""Creates a Node key generator."""
+
+		self.is_imported = bool(import_source)
+		if self.is_imported:
+			self.import_source = Path(import_source).absolute()
+
+	def create_key_file(self, factory):
```

Comment:
create => write / export ?

Reply:
I prefer to ``write`` since it is not always ``export`` of a key.